test/oidc-utils.spec.ts

```diff
@@ -55,36 +20,57 @@ describe('OidcUtils', (): void => {
             oidcAudience: 'aud',
         };
 
+        afterEach((): void => {
+            jest.restoreAllMocks();
```

Comment:
I see we already call `jest.restoreAllMocks()` in the outer `afterEach`. Is it still needed here too?